docs/reference/modules/node.asciidoc

```diff
@@ -354,6 +357,9 @@ To create a dedicated {transform} node in the {default-dist}, set:
 node.roles: [ transform ]
 ----
 
+If you want the node to use {ccs}, add `remote_cluster_client` to the list of
+roles. See <<remote-node>>.
+
```

Comment:
```suggestion

If you are using {ccs} in any way it is strongly recommended to allow {transform} nodes to do remote searches unless you have a very good reason not to. It will likely lead to confusion and frustration if cross cluster searches usually work perfectly but fail when used in {transform} configs.
```